Maskbook/Supporting Files/Coordinator.swift

```diff
@@ -208,6 +208,16 @@ class Coordinator {
         case debug
     }
     
+    func getMaskSocialViewController() -> MaskSocialViewController? {
+        let rootVC = window?.rootViewController
+        if let nav = rootVC as? UINavigationController {
+            return nav.topViewController as? MaskSocialViewController
```

Comment:
In which case `MaskSocialViewController` would be navigation's topViewController?
I think `MaskSocialViewController` can only be rootViewController

Reply:
https://github.com/DimensionDev/Mask-iOS/blob/beda646264db5d0fa869d34306f2128dd49d29cd/Maskbook/Supporting Files/Coordinator.swift#L227